### PR TITLE
put supervisor log files in (assemblroot)/var/log. 

### PR DESCRIPTION
**Pull request by maparent**
Basic size/num backup values. Remove some obsolete values.

### CODE REVIEW

File: production.ini [hunk 1]
```diff
@@ -513,14 +513,19 @@ autostart_celery_notify_beat = true
...
 autostart_urlmetadata = true
+autostart_elasticsearch = true
+autostart_maintenance = false
+sup_log_backups_celery_notify_beat = 0
+sup_log_backups_gulp = 0
+sup_log_backups_webpack = 0
+sup_log_backups_maintenance = 0
+sup_log_backups_uwsgi = 20
+sup_log_backups_pserve = 2
+sup_log_size_uwsgi_err = 4000000
```
Comment (ayyazdaniaryan) [inline production.ini]: Comments of the numbers here won't hurt future devs from digging in supervisor docs to understand that this is in bytes, so this particular is about ~ 3.8 MB.

File: supervisord.conf.tmpl [hunk 2]
```diff
@@ -34,6 +34,12 @@ stopwaitsecs = 60
 # If the process didn't kill it's children after 60 seconds, it's unlikely to
 # ever reap them, so kill them all
 killasgroup = true
+stdout_logfile = %(here)s/var/log/celery.log
+stdout_logfile_maxbytes = %(sup_log_size_celery_out)s
+stdout_logfile_backups = %(sup_log_backups_celery_out)s
+stderr_logfile = %(here)s/var/log/celery.err
```
Comment (ayyazdaniaryan) [inline supervisord.conf.tmpl]: Can't change `%(here)s/var/log` into a variable as well so you don't repeat this path, also gives flexibility on where the logs can go. So, 2 variables to tune, but defaults to /var/log in assembl directory.

File: assembl/scripts/ini_files.py [hunk 3]
```diff
@@ -55,8 +55,8 @@
     'autostart_webpack': 'false',
     'autostart_uwsgi': 'false',
     'autostart_elasticsearch_server': 'false',
-    'autostart_metrics_server': 'false',
-    'autostart_edgesense_server': 'false',
+    'sup_log_size': '1000000',
```
[the remark this answers is not in the document]
Comment (maparent) [reply assembl/scripts/ini_files.py]: correct. could rename size to bytes?

File: assembl/scripts/ini_files.py [hunk 4]
```diff
@@ -126,32 +105,37 @@ def generate_ini_files(config, config_fname):
...
+                name1 = '_'.join((name0, procname))
+                name = '_'.join((name1, stream))
+                try:
+                    val = config.get('supervisor', name)
+                except NoOptionError:
+                    try:
+                        val = config.get('supervisor', name1)
+                    except NoOptionError:
+                        val = config.get('supervisor', name0)
+                vars[name] = val
```
[the remark this answers is not in the document]
Comment (maparent) [reply assembl/scripts/ini_files.py]: the default in that case is given in DEFAULTS above.

**Pull request closed without merging**